.gitignore

```diff
@@ -1,4 +1,3 @@
-dist/
```

Comment:
Yeh, that should be removed. Its only there so I can use the code in one of my projects without publishing to npm